nipype/interfaces/fsl/epi.py

```diff
@@ -659,6 +659,8 @@ class EddyInputSpec(FSLCommandInputSpec):
         "the field specified by --field and first volume "
         "in file --imain")
     use_cuda = traits.Bool(False, desc="Run eddy using cuda gpu")
+    cnr_maps = traits.Bool(False, desc='Output CNR-Maps', argstr='--cnr_maps')
+    residuals = traits.Bool(False, desc='Output Residuals', argstr='--residuals')
```

Comment:
Do we know if these were introduced in a particular version? If so, we should set the `min_ver` flag.

Reply:
I added 5.0.10 as the minimum version since this is the one i am using and that is working (despite the parameters not showing in the usage page). I messaged eddy's developers and they said that these parameters are actually planned to be released with eddy 6.0.1, they did not yet answer why the parameters are already usable in lower versions.